repos/system_upgrade/el7toel8/actors/selinux/selinuxapplycustom/actor.py

```diff
@@ -0,0 +1,115 @@
+import os
+from shutil import rmtree
+
+from leapp.actors import Actor
+from leapp.models import SELinuxModules, SELinuxCustom, SELinuxRequestRPMs, CheckResult
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
+from leapp.libraries.stdlib import run, CalledProcessError
+
+WORKING_DIRECTORY = '/tmp/selinux/'
+
+class SELinuxApplyCustom(Actor):
+    '''
+    Re-apply SELinux customizations from RHEL-7 installation
+
+    Re-apply SELinux policy customizations (custom policy modules and changes
+    introduced by semanage). Any changes (due to incompatiblity with RHEL-8
+    SELinux policy) are reported to user.
+    '''
+    name = 'selinuxapplycustom'
+    consumes = (SELinuxCustom, SELinuxModules, )
+    produces = (CheckResult, )
+    tags = (ApplicationsPhaseTag, IPUWorkflowTag, )
+
+    def process(self):
+        # cil module files need to be extracted to disk in order to be installed
+        try:
+            # clear working directory
+            rmtree(WORKING_DIRECTORY)
+        except OSError:
+            # expected
+            pass
+        try:
+            os.mkdir(WORKING_DIRECTORY)
+        except OSError:
+            self.log.info("Failed to access working directory! Aborting.")
+            return
+
+        # import custom SElinux modules
+        for semodules in self.consume(SELinuxModules):
+            self.log.info("Processing custom SELinux policy modules. Count: %d." % len(semodules.modules))
+            for module in semodules.modules:
+                cil_filename = "%s%s.cil" % (WORKING_DIRECTORY, module.name)
+                self.log.info("Installing module %s on priority %d." % (module.name, module.priority))
```

Comment:
For logging you can avoid applying the arguments immediately and have it done by the logger, in case info wouldn't be logged, it wouldn't perform the string formatting. Premature optimization, I know, but that's how one should use those loggers. Otherwise please prefer .format style string formatting.
```suggestion
                self.log.info("Installing module %s on priority %d.", module.name, module.priority)                
```